code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -452,7 +452,6 @@
 	icon_state = "hardsuit1-sbg"
 	item_state = "hardsuit1-sbg"
 	hardsuit_type = "sbg"
-	armor = list("melee" = 40, "bullet" = 50, "laser" = 30, "energy" = 40, "bomb" = 35, "bio" = 100, "rad" = 50, "fire" = 50, "acid" = 90)
 
 /obj/item/clothing/suit/space/hardsuit/syndi/sbg
 	name = "beige-red hardsuit"
```

Comment:
Edit the Ramzi suit and helmet's description to say it's made by the Ramzi Clique instead of just Gorlex Marauders, since you're already touching this file

Reply:
done !